### PR TITLE
Amanda Robbers Fix

### PR DESCRIPTION
**Pull request by sunhwapark**

### CODE REVIEW

File: EnglishTranslation-master/ETC.tsv [hunk 1]
```diff
@@ -14554,9 +14554,9 @@ ETC_20150918_014555	Collecting
 ETC_20150918_014556	Burning
 ETC_20150918_014557	Found the explosive!
 ETC_20150918_014558	Help the soldier
-ETC_20150918_014559	Find the equipment
-ETC_20150918_014560	You've found an usable breastplate
-ETC_20150918_014561	You've found an usable spear
+ETC_20150918_014559	Searching for equipment
+ETC_20150918_014560	You've found an usable Breastplate
```
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:14558]: 'an usable' or 'a usable' ?

**Comment by sunhwapark** [comment]
I think

**Comment by sunhwapark** [comment]
Updating PR with feedback ( Hit the wrong button closing this by mistake. :P )

**Comment by sunhwapark** [comment]
I'll add one more final update in a few hours.
I'll send a message when ready for another revision.

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**